cmd/modulectl/create/long.txt

```diff
@@ -14,21 +14,32 @@ The module config file is a YAML file used to configure the following attributes
 - name:             a string, required, the name of the module
 - version:          a string, required, the version of the module
 - channel:          a string, required, channel that should be used in the ModuleTemplate CR
-- manifest:         a string, required, reference to the manifest, must be a relative file name or URL
+- manifest:         a string, required, reference to the manifest, must be a URL
 
-- defaultCR:        a string, optional, reference to a YAML file containing the default CR for the module, must be a relative file name or URL
+- defaultCR:        a string, optional, reference to a YAML file containing the default CR for the module, must be a URL
 - mandatory:        a boolean, optional, default=false, indicates whether the module is mandatory to be installed on all clusters
 - resourceName:     a string, optional, default={NAME}-{CHANNEL}, the name for the ModuleTemplate CR that will be created
 - internal:         a boolean, optional, default=false, determines whether the ModuleTemplate CR should have the internal flag or not
 - beta:             a boolean, optional, default=false, determines whether the ModuleTemplate CR should have the beta flag or not
 - security:         a string, optional, name of the security scanners config file
 - labels:           a map with string keys and values, optional, additional labels for the generated ModuleTemplate CR
 - annotations:      a map with string keys and values, optional, additional annotations for the generated ModuleTemplate CR
+- manager:          # an object, optional, module resource that indicates the installation readiness of the module
+    name: a string, required, the name of the module resource
+    namespace: a string, optional, the namespace of the module resource
+    group: a string, required, the API group of the module resource
+    version: a string, required, the API version of the module resource
+    kind: a string, required, the API kind of the module resource
+- associatedResources: a list of Group-Version-Kind(GVK), optional, resources that should be cleaned up with the module deletion
+- resources:        # a map with string keys and values, optional, additional resources of the module that may be fetched
+    - name: a string, required, the name of the resource
+      link: a URL, required, the link to the resource
```

Comment:
Sorry for being stubborn about that... Can the strings that belong to the manager object be also indented in the same way?